.github/workflows/python-uv-release.yaml

```diff
@@ -0,0 +1,127 @@
+        type: string
+
+    secrets:
+      github-username:
+        description: "The GitHub username for committing the changes."
+        required: true
+      github-email:
+        description: "The GitHub email for committing the changes."
+        required: true
+      github-token:
+        description: "The GitHub token for committing the changes."
+        required: true
+
+    # Map the workflow outputs to job outputs
+    outputs:
+      release-version:
+        description: "The bumped version."
+        value: ${{ jobs.release.outputs.release-version }}
+      old-version:
+        description: "The old version in your `.bumpversion.cfg` file."
+        value: ${{ jobs.release.outputs.old-version }}
+
+jobs:
+  release:
+    runs-on: ubuntu-22.04
+    # Map the job outputs to step outputs
+    outputs:
+      release-version: ${{ steps.bump-version-release.outputs.release-version }}
+      old-version: ${{ steps.bump-version-dev.outputs.old-version }}
+
+    steps:
+      - name: Install uv
+        uses: astral-sh/setup-uv@v5
+        with:
+          python-version: ${{ inputs.python-version }}
+          version: ${{ inputs.uv-version }}
+
+      - name: Check out repository
+        uses: bakdata/ci-templates/actions/checkout@1.49.0
+        with:
+          ref: ${{ github.event.repository.default_branch }}
+          persist-credentials: false # required for pushing to protected branch later
+          fetch-depth: 0 # required for changelog generation
+
+      - name: Bump version
+        id: bump-version-dev
+        uses: bakdata/ci-templates/actions/bump-version@v1.21.2
+        with:
+          release-type: ${{ inputs.release-type }}
+          working-directory: ${{ inputs.working-directory }}
+
+      - name: Bump version release
+        id: bump-version-release
+        uses: bakdata/ci-templates/actions/bump-version@v1.21.2
+        with:
+          release-type: release --allow-dirty
+          working-directory: ${{ inputs.working-directory }}
+
+      - name: Apply version bump to uv lock
+        run: uv lock --no-refresh
+
+      - name: Create changelog
+        id: build-changelog
+        uses: bakdata/ci-templates/actions/changelog-generate@1.50.0
+        if: ${{ inputs.changelog }}
+        with:
+          github-token: ${{ secrets.github-token }}
+          tag: ${{ steps.bump-version-release.outputs.release-version }}
+          changelog-file: ${{ inputs.changelog-file }}
+
+      - name: Commit and push changes including .bumpversion.cfg file
+        uses: bakdata/ci-templates/actions/commit-and-push@1.25.2
+        with:
+          ref: ${{ github.event.repository.default_branch }}
+          commit-message: "Bump version ${{ steps.bump-version-dev.outputs.old-version }} → ${{ steps.bump-version-release.outputs.release-version }}"
+          github-username: ${{ secrets.github-username }}
+          github-email: ${{ secrets.github-email }}
+          github-token: ${{ secrets.github-token }}
```

Comment:
Don't you need `add-untracked: "true"` here. See https://github.com/bakdata/ci-templates/blob/1.25.2/actions/commit-and-push/action.yaml#L21-L24

Reply:
good point! added